akka-persistence/src/test/scala/akka/persistence/AtLeastOnceDeliverySpec.scala

```diff
@@ -48,9 +52,13 @@ object AtLeastOnceDeliverySpec {
     var lastSnapshotAskedForBy: Option[ActorRef] = None
 
     def updateState(evt: Evt): Unit = evt match {
+      case AcceptedReq(payload, destination) if actorSelectionDelivery ⇒
+        log.debug(s"deliver(destination, deliveryId ⇒ Action(deliveryId, $payload)), recovery: " + recoveryRunning)
```

Comment:
we should avoid `toString` of arbitrary messages (payload), it might be huge and fill up memory and disk